frontstage/views/passwords/reset_password.py

```diff
@@ -20,11 +20,11 @@ def get_reset_password(token, form_errors=None):
         party_controller.verify_token(token)
     except ApiError as exc:
         if exc.status_code == 409:
-            logger.warning('Token expired', status_code=exc.status_code, token=token)
+            logger.warning('Token expired', url=exc.url, status_code=exc.status_code, token=token)
             return render_template('passwords/password-expired.html')
         elif exc.status_code == 404:
-            logger.warning('Invalid token sent to party service', status_code=exc.status_code, token=token)
-            return redirect(url_for('error_bp.not_found_error_page'))
+            logger.warning('Invalid token sent to party service', url=exc.url, status_code=exc.status_code, token=token)
+            return render_template('errors/404-error.html'), 404
```

Comment:
Instead of calling `render_template` we should call `abort(404)` to use the common error handler so we're not duplicating the template reference.